src/main/target/FLYWOOF7DUAL/target.c

```diff
@@ -29,7 +29,7 @@
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
 
-    DEF_TIM(TIM11, CH1, PB9,  TIM_USE_PPM,   0, 0), // PPM
+    DEF_TIM(TIM8, CH1, PC6,  TIM_USE_PPM, 0, 0),  // PPM(2,2)-TX6
```

Comment:
Why is this being changed? If any boards with the design using PB9 as the PPM pin were sold to customers this will break the LED_STRIP functionality for them.